### PR TITLE
refactor(global_food_explorer): Remove pork and use meat, pig only

### PR DESCRIPTION
**Pull request by pabloarosado**
The [global food explorer](https://ourworldindata.org/explorers/global-food) currently shows "Pork" and "Meat, pig" as separate items, each one with different elements. This is not unreasonable, since they are indeed different items in the original FAO data. However, in practice, for users, they probably mean the same thing, so I suppose it makes sense to put them together.
For consistency, since we have all meat products labeled as "Meat, X", I renamed "Pork" → "Meat, pig".

What I did in this branch was:
* Remove the line of "pork" in the file `foods.csv`.
* Add the tags that used to be in "pork" to the tags of "Meat, pig".
* Run the `global-food-explorer.py` script to update the `global-food.explorer.tsv` explorer file.

**NOTE**: This PR should only be merged after the ETL explorers data has been updated in master (the corresponding PR doesn't exist yet).

### CODE REVIEW

File: scripts/global-food-explorer/foods.csv [hunk 1]
```diff
@@ -97,7 +97,7 @@ meat_goose_and_guinea_fowl,"Meat, goose and guinea fowl",Goose and guinea fowl m
 meat_horse,"Meat, horse",Horse meat,Horse meat,production animals_slaughtered animal_yield,
 meat_lamb_and_mutton,"Meat, lamb and mutton",Lamb and mutton meat,Lamb and mutton meat,production animals_slaughtered animal_yield,
 meat_mule,"Meat, mule",Mule meat,Mule meat,production animals_slaughtered animal_yield,
-meat_pig,"Meat, pig",Pig meat,Pig meat,production animals_slaughtered animal_yield,
+meat_pig,"Meat, pig",Pig meat,Pig meat,production animals_slaughtered animal_yield food_balances waste,
```
Comment (marcelgerber) [inline scripts/global-food-explorer/foods.csv:100]: nit: to be consistent with other lines here, it should probably be 
```suggestion
meat_pig,"Meat, pig",Pig meat,Pig meat,production animals_slaughtered food_balances waste animal_yield,
```
in that order. Not sure right now whether it makes a difference in generation, but at least it's consistent then.

**Review by marcelgerber — APPROVED** [review]

**Comment by marcelgerber** [comment]
I'll also mark this as a draft for now, so we don't accidentally merge it before the other change is ready.

File: scripts/global-food-explorer/foods.csv (hunk 1 above)
Comment (pabloarosado) [reply scripts/global-food-explorer/foods.csv:100]: Thanks Marcel. Currently the tags don't seem to always be in the same order, if you compare:
```
meat_total,"Meat, Total",Total meat,All meat,production animals_slaughtered food_balances waste animal_feed,
meat_poultry,"Meat, poultry",Poultry meat,Poultry meat,production animals_slaughtered animal_yield food_balances waste animal_feed,
```
So I guess it doesn't matter? If you think it does, I can sort them all alphabetically, for example.

Comment (marcelgerber) [reply scripts/global-food-explorer/foods.csv:100]: ah, in this case they actually are in the same order, no? It's just that the second line includes `animal_yield`, whereas the first one doesn't.

the order of these tags (somewhat) determines the display order in the explorer, so better to keep the order as-is and change them up as I said above.

Comment (marcelgerber) [reply scripts/global-food-explorer/foods.csv:100]: Ahh, I see now! all good then.
And sorry for the confusion :)

**Pull request merged**